src/parse/string/shift/operand/operand.ts

```diff
@@ -15,6 +19,10 @@ export const parseOperand = (s: DynamicState): void =>
         ? parseEnclosed(s, s.scanner.shift())
         : s.scanner.lookaheadIsIn(Scanner.whiteSpaceTokens)
         ? parseOperand(s.shiftedByOne())
+        : s.scanner.lookahead === "d"
+        ? s.shiftedByOne().scanner.lookaheadIsIn(enclosingChar)
```

Comment:
We need to look specifically for a quote here, which is maybe why you had that separate out 😅

Maybe something like `(s.scanner.shift(), s.scanner.lookaheadIsIn(enclosingQuotes))` would allow you to not cast on line 24